manifests/params.pp

```diff
@@ -91,28 +91,29 @@
 # Copyright 2011 Puppet Labs, unless otherwise noted
 #
 class mrepo::params (
-  $src_root       = '/var/mrepo',
-  $www_root       = '/var/www/mrepo',
-  $www_servername = 'mrepo',
-  $www_ip         = $::ipaddress,
-  $www_ip_based   = false,
-  $user           = 'apache',
-  $group          = 'apache',
-  $source         = 'package',
-  $ensure_src     = 'latest',
-  $selinux        = undef,
-  $rhn            = false,
-  $rhn_config     = false,
-  $rhn_username   = '',
-  $rhn_password   = '',
-  $genid_command  = '/usr/bin/gensystemid',
-  $mailto         = 'UNSET',
-  $git_proto      = 'git',
-  $descriptions   = {},
-  $http_proxy     = '',
-  $https_proxy    = '',
-  $priority       = '10',
-  $port           = '80',
+  $src_root            = '/var/mrepo',
+  $www_root            = '/var/www/mrepo',
+  $www_servername      = 'mrepo',
+  $www_ip              = $::ipaddress,
+  $www_ip_based        = false,
+  $user                = 'apache',
+  $group               = 'apache',
+  $source              = 'package',
+  $ensure_src          = 'latest',
+  $selinux             = undef,
+  $rhn                 = false,
+  $rhn_config          = false,
+  $rhn_username        = '',
+  $rhn_password        = '',
+  $genid_command       = '/usr/bin/gensystemid',
+  $mailto              = 'UNSET',
+  $git_proto           = 'git',
+  $descriptions        = {},
+  $http_proxy          = '',
+  $https_proxy         = '',
+  $priority            = '10',
+  $port                = '80',
+  $createrepo_options  = '',
```

Comment:
the expectation is that this will be a plain string, containing a bunch of options?

```
--some-opt --another-opt -q -p
```

?